roles/nginx/files/fastcgi_cache.conf

```diff
@@ -0,0 +1,15 @@
+set $skip_cache 0;
+
+if ($query_string != "") {
+  set $skip_cache 1;
+}
+
+# Don't cache uris containing the following segments
+if ($request_uri ~* "/wp-admin/|/xmlrpc.php|wp-.*.php|/feed/|index.php|sitemap(_index)?.xml") {
+  set $skip_cache 1;
+}
+
+# Don't use the cache for logged in users or recent commenters
+if ($http_cookie ~* "comment_author|wordpress_[a-f0-9]+|wp-postpass|wordpress_no_cache|wordpress_logged_in") {
```

Comment:
I think this string (and the above for require uri) should be stored in defaults as a variable. Then they can easily be overridden.